app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :post, class_name: 'Post', foreign_key: 'post_id'
+  has_many :comment, class_name: 'Comment', foreign_key: 'author_id', dependent: :destroy
+  has_many :like, class_name: 'Like', foreign_key: 'author_id', dependent: :destroy
+
```

Comment:
- [ ] Current_user method is missing in _app/controllers/application_controller.rb_. Kindly ensure to define the same in  _app/controllers/application_controller.rb_ smilar to:
```ruby
# app/controllers/application_controller.rb
...

def current_user
  User.first
end
```

Reply:
Hi @rloterh

Thank you for the feedback. I just want to clarify a doubt. Please what's the use of the current_user method. 

Thank you

Reply:
@charlesgobina 
Just as the name is, the current user points to the particular user who is presently logged in to the said application at a said point in time